stac_nb/stac_nb.py

```diff
@@ -115,7 +139,7 @@ def click_button(self, b):
 
         query_response = self.client.search(**payload_dict)
 
-        self.query_results = list(query_response.get_items())
+        self.query_results = VisualList(query_response.get_items())
```

Comment:
Ah, bummer.
Well, then you loose the abilities of the PySTAC Items.
I guess it would make more sense to convert to dict inside of Visual Dict's _repr_html_...